src/section/section_members.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright 2020 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use super::{
+    elders_info::EldersInfo,
+    member_info::{AgeCounter, MemberInfo, MemberState, MIN_AGE_COUNTER},
+};
+use crate::{
+    id::{P2pNode, PublicId},
+    xor_space::{Prefix, XorName},
+};
+use itertools::Itertools;
+use std::{
+    cmp::Ordering,
+    collections::{btree_map::Entry, BTreeMap},
+    mem,
+    net::SocketAddr,
+};
+
+/// Container for storing information about members of our section.
+#[derive(Debug, Eq, Serialize, Deserialize)]
+pub struct SectionMembers {
+    members: BTreeMap<XorName, MemberInfo>,
+    // Number that gets incremented every time a node joins or leaves our section - that is, every
```

Comment:
incremented  -> increased ?

Reply:
Same thing, no? But I'll change it if you prefer.

Reply:
Actually, I think "increment" usually means "increase by one" which is what's going on here. So I'd keep it unless you strongly disagree.

Reply:
It's just `increment` is a noun, not a verb. :(
I won't insist on it, so up to you then.  :)

Reply:
It is verb as well: https://en.wiktionary.org/wiki/increment#Verb